### PR TITLE
update list of core-committers

### PR DESCRIPTION
**Pull request by lieut-data**
As discussed in the developers meeting today.

### CODE REVIEW

File: source/developer/contribution-guide.md [hunk 1]
```diff
@@ -92,7 +92,7 @@ If you've included your mailing address in the signed [Contributor License Agree
 
 Core committers on Mattermost repositories consist of vetted core team members, including both community contributors as well as staff from Mattermost, Inc., who are trusted to review and merge PRs.
 
-- **Core committers include**: ccbrown, coreyhulen, cpanato, crspeller, csduarte, enahum, grundleborg, hmhealey, jespino, jwilander, mkraft, MusikPolice, saturninoabril, stephenkiers
+- **Core committers include**: ccbrown, coreyhulen, cpanato, crspeller, csduarte, enahum, grundleborg, hmhealey, jespino, jwilander, lieut-data, mkraft, saturninoabril
```
Comment (lieut-data) [inline source/developer/contribution-guide.md]: Maintaining alphabetical order.

File: source/developer/core-developer-handbook.md [hunk 2]
```diff
@@ -38,18 +38,15 @@ Below is the list of core developers working on Mattermost:
...
   - @martin.kraft on [pre-release.mattermost.com](https://pre-release.mattermost.com/core/messages/@martin.kraft) and [@mkraft](https://github.com/mkraft) on GitHub
   - Dev areas: All areas
 - **Jesús Espino**
   - @jesus.espino on [pre-release.mattermost.com](https://pre-release.mattermost.com/core/messages/@jesus.espino) and [@jespino](https://github.com/jespino) on GitHub
   - Dev areas: Webapp, Permissions and Compliance, Sample data generation
-- **Stephen Kiers**
-  - @stephen.kiers on [pre-release.mattermost.com](https://pre-release.mattermost.com/core/messages/@stephen.kiers) and [@stephenkiers](https://github.com/stephenkiers) on GitHub
-  - Dev areas: Webapp, Mobile UX and UI
+- **Jesse Hallam**
+  - @jesse.hallam on [pre-release.mattermost.com](https://pre-release.mattermost.com/core/messages/@ccbrown) and [@lieut-data](https://github.com/ccbrown) on GitHub
```
Comment (jasonblais) [inline source/developer/core-developer-handbook.md]: `ccbrown` is in the two links

**Comment by coreyhulen** [comment]
Feel free to merge once minor mistake is fixed.

**Comment by lieut-data** [comment]
Gah, yep. Thanks!

**Pull request merged**